migrations/2017-09-26-200549_users_lowercase_index/up.sql

```diff
@@ -0,0 +1 @@
+CREATE UNIQUE INDEX lower_gh_login ON users (LOWER(gh_login));
```

Comment:
Should we get rid of `index_users_gh_login` since it's redundant?

Reply:
Probably, or just change that index to lower(gh_login) instead of using this index. Will need to address the UNIQUE issue though. Don't know enough about this app to tell if it's the way to unit tests are laid out or if it could be a production issue.

Reply:
If it's a production issue we'll deal with it